### DISCUSSION TITLE
How to make the lighting effect the same?

### DISCUSSION THREAD

**Discussion by amin2312** [jnsmalm/pixi3d#154] [Q&A]
Hi, sir.

let light = new PIXI3D.Light()
light.type = "directional";
light.rotationQuaternion.setEulerAngles(0, 180, 0);
PIXI3D.LightingEnvironment.main.lights.push(light);
PIXI3D.Camera.main.orthographic = true;
let mesh;
mesh = StageIns.addChild(PIXI3D.Mesh3D.createSphere())
mesh.material.baseColorTexture = PIXI.Texture.from("101.png");

mesh = StageIns.addChild(PIXI3D.Mesh3D.createSphere())
mesh.material.baseColorTexture = PIXI.Texture.from("101.png");
mesh.position.set(-3,0,0);

mesh = StageIns.addChild(PIXI3D.Mesh3D.createSphere())
mesh.material.baseColorTexture = PIXI.Texture.from("101.png");
mesh.position.set(3,0,0);

![3ball](https://user-images.githubusercontent.com/4578021/212054683-52c6fc8f-93c5-450f-85c4-88a4e787883b.jpg)

In orthographic camera mode
hwo to let the left ball and right ball same as the middle ball? the light effect

**Comment by jnsmalm** [comment]
You can use local lighting environments.

```
app.loader.load((_, resources) => {
  let mesh1 = app.stage.addChild(createSphereWithLocalPointLight())
  mesh1.x = 2

  let mesh2 = app.stage.addChild(createSphereWithLocalPointLight())
  mesh2.x = -2
})

function createSphereWithLocalPointLight() {
  let mesh = app.stage.addChild(PIXI3D.Mesh3D.createSphere())
  
  let pointLight = mesh.addChild(new PIXI3D.Light())
  pointLight.z = 5

  mesh.material.lightingEnvironment = new PIXI3D.LightingEnvironment(app.renderer)
  mesh.material.lightingEnvironment.lights.push(pointLight)

  return mesh
}
```
